src/io/xml_schemas/vtu_file/vtu_file_schema.xsd

```diff
@@ -56,74 +56,70 @@ copyright : not copyrighted - public domain
       <xsd:element name="PointData">
         <xsd:complexType>
           <xsd:sequence>
-            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded"/>
+            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded" />
           </xsd:sequence>
         </xsd:complexType>
       </xsd:element>
       <xsd:element name="CellData">
         <xsd:complexType>
           <xsd:sequence>
-            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded"/>
+            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded" />
           </xsd:sequence>
         </xsd:complexType>
       </xsd:element>
       <xsd:element name="Points">
         <xsd:complexType>
           <xsd:sequence>
-            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded"/>
+            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded" />
           </xsd:sequence>
         </xsd:complexType>
       </xsd:element>
       <xsd:element name="Cells">
         <xsd:complexType>
           <xsd:sequence>
-            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded"/>
+            <xsd:element name="DataArray" type="DataArray_t" minOccurs="0" maxOccurs="unbounded" />
           </xsd:sequence>
         </xsd:complexType>
       </xsd:element>
-  	</xsd:sequence>
-  	<xsd:attribute name="NumberOfPoints" type="xsd:integer" use="required"/>
-  	<xsd:attribute name="NumberOfCells" type="xsd:integer" use="required"/>
-  </xsd:complexType>	
+    </xsd:sequence>
+    <xsd:attribute name="NumberOfPoints" type="xsd:integer" use="required" />
+    <xsd:attribute name="NumberOfCells" type="xsd:integer" use="required" />
+  </xsd:complexType>
 
   <xsd:complexType name="UnstructuredGrid_t">
     <xsd:all>
-      <xsd:element name="Piece" type="PieceUnstructuredGrid_t"/>
+      <xsd:element name="Piece" type="PieceUnstructuredGrid_t" />
     </xsd:all>
   </xsd:complexType>
 
   <xsd:complexType name="PolyData_t">
```

Comment:
I think this polydata datatype is never used for our project. Should be removed.